scss/civicrm/dashboard/pages/_view-dashboard.scss

```diff
@@ -0,0 +1,233 @@
+      }
+    }
+
+    [id*="civicrm-news-feed-"] {
+      padding: 0 !important;
+
+      .crm-accordion-wrapper {
+        &.collapsed .crm-accordion-header:before {
+          color: $gray-darker;
+          content: $fa-caret-right;
+          font-size: $font-size-base;
+        }
+
+        &:not(.collapsed) .crm-accordion-header:before {
+          color: $gray-darker;
+          content: $fa-caret-down;
+          font-size: $font-size-base;
+        }
+      }
+
+      .crm-accordion-header {
+        background: $crm-white !important;
+        border-bottom: 1px solid $crm-dark-grayblue;
+
+        .crm-news-feed-item-preview,
+        .crm-news-feed-item-title {
+          color: $crm-dasboard-header;
+          font-family: $font-family-base;
+          font-weight: $crm-font-weight-h1 !important;
+        }
+      }
+
+      .crm-accordion-body {
+        box-shadow: $crm-box-shadow-inset;
+        color: $crm-copy;
+        font-family: $font-family-base;
+        font-size: $font-size-base;
+        padding: 10px 15px !important;
+
+        .crm-news-feed-item-link {
+          text-transform: capitalize;
+        }
+      }
+    }
+  }
+
+  .crm-activity-selector-dashlet {
+    display: grid;
+
+    .crm-accordion-header {
+      border-bottom: 1px solid $crm-background;
+      color: $gray-darker;
+      font-weight: $crm-font-weight-h1 !important;
+    }
+
+    .crm-search_filters-accordion {
+      margin-bottom: 0;
+
+      .crm-accordion-body {
+        padding: 20px 15px !important;
+      }
+
+      .crm-inline-edit-field {
+        float: left;
+        margin-bottom: 20px;
+        width: 50%;
+
+        .select2-container {
+          width: 98% !important;
+        }
+
+        label {
+          display: block;
+          margin-bottom: 5px;
+        }
+      }
+    }
+
+    .contact-activity-selector-dashlet {
+      
+      tr td,
+      tr th {
+        padding: 10px;
+      }
+    }
+
+    .dataTables_wrapper {
+      box-shadow: none;
+      border-radius: 0;
+      margin-bottom: 0;
+
+      table {
+        box-shadow: none;
+        width: 100% !important;
+      }
+
+      .crm-datatable-pager-top {
+        border-radius: 0;
+      }
+
+      .crm-datatable-pager-bottom {
+        padding: 0 15px;
+        line-height: 1.5em;
+      }
+
+      .dataTables_paginate {
+        width: 50% !important;
+      }
+
+      .dataTables_paginate .ui-button {
+        padding: 10px;
+      }
+
+      .dataTables_info {
+        padding: 10px 0;
+        width: 49%;
+      }
+
+      div[id*="s2id_autogen"] {
+        .select2-choice {
+          background: none;
+          border: 1px solid $crm-darker-grayblue;
+          border-radius: $border-radius-small;
+          box-shadow: $crm-select2-box-shadow-inset;
+        }
+
+        .select2-arrow {
+          background: none;
+          padding: 0 5px;
+        }
+      }
+    }
+  }
+
+  #civicrm-getting-started {
+
+    h3 {
```

Comment:
is this styling of `h3` generic only to `getting-started`? Or is it how `h3` should appear everywhere in civicrm

Reply:
What about this?

Reply:
@AkA84 This is for this specific section not for everywhere in global